src/js/steps/initMonkey.js

```diff
@@ -21,6 +23,26 @@ module.exports = function ($events, options) {
 
     data.monkeyContainer.addClass('ready');
 
+    // Track errors on image load
+    var $images = data.monkeyContainer.find('[class*="page"] img');
+    $images.on('error', function () {
+      if (window.analytics) {
+        analytics.track('Broken image in monkey', { src: this.src });
+      }
+    });
+
+    var remaining = $images.filter(function () {
+      return !this.complete;
+    }).length;
+
+    $images.on('load', function () {
+      remaining--;
+
+      if (!remaining && window.analytics && options.platformAPI) {
+        analytics.track('Monkey images all loaded');
```

Comment:
It's possible, but I don't think it'll be useful data. The slowest cases will be users on bad connections, not the creative platform being slow. Also, we intentionally delay the loading of some images.